### PR TITLE
Cleared up sentence and removed duplicate text.

### PR DESCRIPTION
**Pull request by wesleyfsmith**
I think the readme is clearer this way than having the same sentence twice.

### CODE REVIEW

**Comment by mitar** [comment]
Thanks! Please sign CLA and I can merge this.

**Comment by wesleyfsmith** [comment]
I signed the CLA but I'm not sure why the tests timed out.

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Here are two Spacebars templates from an example app called "Leaderboard" which
 </template>
 ```
 
-The template tags `{{name}}` and `{{score}}` refer to properties of the data context (the current player), while `players` and `selected` refer to helper functions.  Helper functions and event handlers are defined in JavaScript:
+The template tags `{{name}}` and `{{score}}` refer to properties of the data context (the current player), while `players` and `selected` refer to helper functions.
```
Comment (mitar) [inline README.md]: Can you in both cases not use `{{` and `}}` around? Because it is not consistent. For example, it is `{{selected}}` in the example.

**Comment by mitar** [comment]
Ignore timeouts. These are some issues with testing (#62). I was not able to fix it.

**Pull request merged**